README.md

```diff
@@ -124,6 +147,10 @@ accordingly.
 
 ### preauth-unit-time-booking:
 
+**DEPRECATED** The current default process is named
+  `flex-default-process` and it can perform the
+  `preauth-unit-time-booking` functionality.
+
```

Comment:
In the default process the booking creation is day-based, so out of the box it does not replace this process. Should we add another default process that has time-based bookings? Then the default day-based bookings process could ship with ftw-daily and the time-based process with ftw-hourly?

Another option is that we keep only one default process here and rely on documentation to guide users to update the `create-booking` action config if they take it into use. But a default process here for time-based bookings would map nicely to the example processes in [ftw-hourly](https://github.com/sharetribe/ftw-hourly/blob/master/ext/transaction-process/process.edn) and [ftw-daily](https://github.com/sharetribe/ftw-daily/blob/master/ext/transaction-process/process.edn) repos.

Reply:
I'll update the docstring to depict this situation. Let's keep only one default process. Good catch!